### ISSUE TITLE
Datenquelle wird verworfen.

### ISSUE THREAD

**Issue by Maxtor62** [SKB-CGN/ioBroker.energiefluss-erweitert#11]
Bei der Einstellung Rectangle>Source geht der Datenpung (Source) verloren.

**Comment by Maxtor62** [comment]
Wenn ich die Seite das nächste mal wieder öffne ist die Füllung weg.

**Comment by SKB-CGN** [comment]
Füllung in der Live-Ansicht oder Config?

**Comment by Maxtor62** [comment]
Sowohl als auch. Meistens wenn man etwas anderes einstellt.

**Comment by SKB-CGN** [comment]
Kannst Du mir die Schritte bitte einmal per Bilder zeigen oder Gif Animation?

**Comment by Maxtor62** [comment]
Also, ich stelle den source für die Batterie ein
![Source](https://user-images.githubusercontent.com/100165416/236450244-f7ed7454-a1a5-4a1c-8927-197468d2419e.PNG)
mache Save>Save an Exit.
![source-speicher](https://user-images.githubusercontent.com/100165416/236450374-89bbc64f-401b-47c5-8a89-02468d56536e.PNG)
Baue ein weiteres Element ein
![neues-element](https://user-images.githubusercontent.com/100165416/236450467-d897c568-5fcf-48e1-b2c3-4767c5cd0204.PNG)
Mache Save>Save and Exit.
![source-speicher-2](https://user-images.githubusercontent.com/100165416/236450593-4fb15b14-b2ed-4d90-afd1-5efc2542a2de.PNG)
Die Batterie ist nicht gefüllt und
![source-weg](https://user-images.githubusercontent.com/100165416/236450665-3955a8a0-c92b-429b-92b3-72e04b147778.PNG)
die Source Einstellung ist weg.

**Comment by SKB-CGN** [comment]
Hab's gefunden - kannst Du das mal wie folgt gegenprüfen?

Du möchtest sicherlich nur die Füllfarbe haben, richtig? Nicht abhängig von einem Wert?

Mal mit F12 im Browser gucken, wie oft dort:

Fill-Gradient added for: XX

erscheint? Dies dann in Anhägigkeit der Anzahl der Elemente, die eine Farbe haben sollen?

**Comment by Maxtor62** [comment]
Nein, ich wollte die Füllung abhängig vom Wert der Batterieladung. Sonst bräuchte ich ja keine Source angeben. Oder habe ich was falsch verstanden?

Eine Füllu
![fuellung](https://user-images.githubusercontent.com/100165416/236474289-8b588a0a-f843-4e6a-97a0-439f7d1ddaca.PNG)
ng ohne Value Wert macht man doch so, oder?

**Comment by SKB-CGN** [comment]
Alles gut. Ist richtig, wie du es machst 👍

**Comment by Maxtor62** [comment]
Nein, die bleibt nicht. Nach jedem Element oder Text, das ich hinzufüge ist die Source wieder weg.
